src/storage/src/hummock/cache.rs

```diff
@@ -573,6 +597,11 @@ impl<K: LruKey, T: LruValue> LruCache<K, T> {
         drop(data);
     }
 
+    unsafe fn add_ref(&self, handle: *mut LruHandle<K, T>) {
```

Comment:
May want to `AtomicUsize` for reference count like `Arc` to avoid locking here

Reply:
no. You can see that whether we use atomicusize or not, we must lock the shard because of other operations

Reply:
If we use atomicusize, the clone of cache entry don't need to acquire the lock.